### PR TITLE
NIFI-12502: Handle additional Property Table capabilities

### PR DESCRIPTION
**Pull request by mcgilman**
NIFI-12502:
- Go To Service.
- Save before navigating if form is dirty.
- Go To Parameter.
- Convert To Parameter.

### CODE REVIEW

**Comment by rfellows** [comment]
Will review...

**Review by rfellows — COMMENTED** [review]
A few comments

File: nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-frontend/src/main/nifi/src/app/pages/flow-designer/state/controller-services/controller-services.effects.ts [hunk 1]
```diff
@@ -234,18 +253,134 @@ export class ControllerServicesEffects {
...
+                                        })
+                                    );
+                                }),
+                                catchError((error) => {
+                                    // TODO handle error
+                                    return NEVER;
+                                })
+                            );
+                        };
+                    }
```
[the remark this answers is not in the document]
Comment (mcgilman) [reply nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-frontend/src/main/nifi/src/app/pages/flow-designer/state/controller-services/controller-services.effects.ts:367]: Thanks for calling this out. I agree that we should investigate ways to improve this. The block is identical but does reference a method that is different and the dialog instances that are different. From an offline discussion, we will investigate opportunities to pull some of this logic into separate services allowing the Effects to be more concise and helpful with some re-use in follow on efforts.

**Review by rfellows — APPROVED** [review]
Latest changes look good to me. Thanks @mcgilman!

**Pull request merged**